experiments/ClimaCore/atm-ocn-lnd/atmos_simulation.jl

```diff
@@ -123,8 +122,12 @@ function atmos_simulation(
 
     # Put all prognostic variable arrays into a vector and ensure that solve can partition them
     Y_atm = ArrayPartition((T_atm_0.Yc, T_atm_0.Yf, zeros(3)))
-    prob_atm = ODEProblem(∑tendencies_atm!, Y_atm, (start_time, stop_time), (parameters, [land_surface_temperature]))
-    simulation = init(prob_atm, SSPRK33(), dt = Δt_min, saveat = 1 * Δt_min)
+
+    ode_algo = CTS.ExplicitAlgorithm(CTS.RK4())
```

Comment:
Here you are changing the algorithm. Is RK4 the best one to use?

Reply:
(Same below)

Reply:
I changed to RK4 because it's an `ExplicitAlgorithm` in ClimaTimeSteppers, whereas SSP333 is an `IMEXAlgorithm` ([docs](https://clima.github.io/ClimaTimeSteppers.jl/dev/api/ode_solvers/#ClimaTimeSteppers.IMEXAlgorithm)). The interface for `ExplicitAlgorithm` is simpler (doesn't require specifying Newton's method args), and we're stepping everything explicitly anyway so I thought this was a simpler change. The simulations in `experiments/ClimaCore` are pretty simple/short so the timestepper shouldn't make much of a difference

Reply:
Maybe you can have the simulations output checked before merging this?

Reply:
This simulation doesn't even run at the moment, which I need to fix next week. The other simulations that I've changed are tested on buildkite and they run without error and the outputs (energy conservation) look good (no noticeable difference from the previous algorithm)